src/editorial/web/components/lines/styles.ts

```diff
@@ -0,0 +1,41 @@
+import { css } from "@emotion/core"
+import { squigglyImage } from "./squiggly"
+import { dottedImage } from "./dotted"
+import { neutral } from "@guardian/src-foundations"
+
+const neutralBorder = (pillar: Pillar) => {
+	switch (pillar) {
+		case "labs":
+			return neutral[60] // 'dark' theme
+		default:
+			return neutral[86]
+	}
+}
+
+export const lineStyles = (pillar: Pillar, count: 4 | 8) => css`
+	background-image: repeating-linear-gradient(
+		to bottom,
+		${neutralBorder(pillar)},
+		${neutralBorder(pillar)} 1px,
+		transparent 1px,
+		transparent 4px
+	);
+	background-repeat: repeat-x;
+	background-position: top;
+	background-size: 1px ${count === 4 ? "13px" : "29px"};
+	height: ${count === 4 ? "13px" : "29px"};
```

Comment:
Does it make sense to use `rem` here for scaling purposes?

Reply:
Yeah, happy to tinker with this for sure. Perhaps we can address the implementation details of this component in a future PR?

Reply:
Worth tracking in an issue?

Reply:
Sure thing! Feel free to raise one, maybe after this is merged